release-notes/unreleased/IGAPP-373-add-search-query-to-url.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: 373
+show_in_stores: true
```

Comment:
JFYI: As there are no stores on web, you can just set this to false and omit the german release note text below :)

Reply:
Yeah i was wondering about that already. Thanks for the information :)